content/guides/tooling/IDE-integration.md

```diff
@@ -71,6 +71,8 @@ RubyMine, and WebStorm.
 
 ## Intelligent Code Completion
 
+::include{file=partials/warning-cypress-json.md}
```

Comment:
I think this should go after the `Configuration` header so it appears here: https://deploy-preview-4136--cypress-docs.netlify.app/guides/tooling/IDE-integration#Configuration

Reply:
Sure, the Jira ticket mentioned the Intelligent Code Completion location. Update is building...